### PR TITLE
Auth Code with PKCE updates

### PR DESCRIPTION
**Pull request by susanharper-okta**
## Description:
- **What's changed?** 
- **Is this PR related to a Monolith release?** no

### Resolves:

* [OKTA-666914](https://oktainc.atlassian.net/browse/OKTA-666914)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/authcodepkce/examples.md](https://github.com/okta/okta-developer-docs/blob/5a410a394576fb27c637cb8fe893b8c3c3437b4e/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fimplement-grant-type%2Fmain%2Fauthcodepkce%2Fexamples.md) | [76](https://okta.acrolinx.cloud/api/v1/checking/scorecards/ecf82a5e-68ed-4581-b9cf-020daa6129d1) | 28  | 28  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/authcodepkce/flow-diagram.md](https://github.com/okta/okta-developer-docs/blob/5a410a394576fb27c637cb8fe893b8c3c3437b4e/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fimplement-grant-type%2Fmain%2Fauthcodepkce%2Fflow-diagram.md) | [89](https://okta.acrolinx.cloud/api/v1/checking/scorecards/dcb4b43b-463a-45e0-98cf-56a8c0853ce6) | 35  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/authcodepkce/install-sdk.md](https://github.com/okta/okta-developer-docs/blob/5a410a394576fb27c637cb8fe893b8c3c3437b4e/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fimplement-grant-type%2Fmain%2Fauthcodepkce%2Finstall-sdk.md) | [100](https://okta.acrolinx.cloud/api/v1/checking/scorecards/82929e55-a0f9-4c55-af73-c0bd12b0355c) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/authcodepkce/overview.md](https://github.com/okta/okta-developer-docs/blob/5a410a394576fb27c637cb8fe893b8c3c3437b4e/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fimplement-grant-type%2Fmain%2Fauthcodepkce%2Foverview.md) | [85](https://okta.acrolinx.cloud/api/v1/checking/scorecards/1630c019-e0f4-4996-9858-e70cec2b0ce2) | 29  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/authcodepkce/setup-app.md](https://github.com/okta/okta-developer-docs/blob/5a410a394576fb27c637cb8fe893b8c3c3437b4e/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fimplement-grant-type%2Fmain%2Fauthcodepkce%2Fsetup-app.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/6437bfca-3f36-4176-a091-9d35e434681e) | 40  | 72  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/authcodepkce/use-flow.md](https://github.com/okta/okta-developer-docs/blob/5a410a394576fb27c637cb8fe893b8c3c3437b4e/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fimplement-grant-type%2Fmain%2Fauthcodepkce%2Fuse-flow.md) | [88](https://okta.acrolinx.cloud/api/v1/checking/scorecards/95d69c6b-2632-4612-910f-44990d05ab0c) | 41  | 100  | 100  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.7aa8247c-8192-41f2-8081-1097455df220?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/authcodepkce/examples.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
 ### Examples
 
-The following examples present mobile and single-page applications implementing the Authorization Code with PKCE flow to authenticate a user. These are complete example applications that show the entire experience. You can clone each project and run them locally.
+The following examples present mobile and single-page apps implementing the Authorization Code with PKCE flow to authenticate a user. These are complete example applications that show the entire experience. You can clone each project and run them locally.
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/authcodepkce/examples.md:3]: Does authorization code need to be capitalized?

File: packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/authcodepkce/use-flow.md [hunk 2]
```diff
@@ -70,11 +70,11 @@ curl --request POST \
 
 Note the parameters that are being passed:
 
-* `grant_type` is `authorization_code`, indicating that you're using the Authorization Code grant type.
-* `redirect_uri` is the URI that was used to get the authorization code.
-* `code` is the authorization code that you got from the `/authorize` endpoint.
-* `code_verifier` is the PKCE code verifier that your app generated at the beginning of this flow.
-* `client_id` is the client ID of the app integration that you created earlier.
+* `grant_type=authorization_code`: Indicates that you're using the Authorization Code grant type
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/authcodepkce/use-flow.md:73]: Same authorization code question

File: packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/authcodepkce/use-flow.md [hunk 3]
```diff
@@ -70,11 +70,11 @@ curl --request POST \
...
+* `redirect_uri`: The URI that was used to get the authorization code
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/authcodepkce/use-flow.md:74]: Look, it's lowercase here! Can we use lowercase?

**Review by barbaravo-okta — DISMISSED** [review]

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/authcodepkce/examples.md](https://github.com/okta/okta-developer-docs/blob/78ac44c84442e27b1e37a9f5e5306b29253feb85/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fimplement-grant-type%2Fmain%2Fauthcodepkce%2Fexamples.md) | [86](https://okta.acrolinx.cloud/api/v1/checking/scorecards/667b0c61-7a39-4151-bfb5-76ba13622459) | 27  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/authcodepkce/flow-diagram.md](https://github.com/okta/okta-developer-docs/blob/78ac44c84442e27b1e37a9f5e5306b29253feb85/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fimplement-grant-type%2Fmain%2Fauthcodepkce%2Fflow-diagram.md) | [89](https://okta.acrolinx.cloud/api/v1/checking/scorecards/20dc63e5-bb42-421e-ac09-74713d6cf92c) | 35  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/authcodepkce/install-sdk.md](https://github.com/okta/okta-developer-docs/blob/78ac44c84442e27b1e37a9f5e5306b29253feb85/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fimplement-grant-type%2Fmain%2Fauthcodepkce%2Finstall-sdk.md) | [100](https://okta.acrolinx.cloud/api/v1/checking/scorecards/8df36f7b-693d-44ad-87f8-962d4ddca6d8) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/authcodepkce/overview.md](https://github.com/okta/okta-developer-docs/blob/78ac44c84442e27b1e37a9f5e5306b29253feb85/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fimplement-grant-type%2Fmain%2Fauthcodepkce%2Foverview.md) | [85](https://okta.acrolinx.cloud/api/v1/checking/scorecards/9cc2ca20-ce75-4396-abb1-78ed767d1c29) | 29  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/authcodepkce/setup-app.md](https://github.com/okta/okta-developer-docs/blob/78ac44c84442e27b1e37a9f5e5306b29253feb85/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fimplement-grant-type%2Fmain%2Fauthcodepkce%2Fsetup-app.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/16079d58-3cc6-46d4-98a4-a12982990532) | 40  | 72  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/authcodepkce/use-flow.md](https://github.com/okta/okta-developer-docs/blob/78ac44c84442e27b1e37a9f5e5306b29253feb85/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fimplement-grant-type%2Fmain%2Fauthcodepkce%2Fuse-flow.md) | [88](https://okta.acrolinx.cloud/api/v1/checking/scorecards/6a3970f2-a7db-4fa7-9b86-d1c544a74a95) | 41  | 100  | 100  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.9a941cde-d02a-418c-9627-ec2717eb4a37?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by barbaravo-okta — APPROVED** [review]

**Pull request merged**